satpy/writers/mitiff.py

```diff
@@ -563,7 +571,7 @@ def _make_image_description(self, datasets, **kwargs):
 
         _image_description += " Map projection: Stereographic\n"
 
-        _image_description += self._add_proj4_string(datasets, first_dataset)
+        _image_description += self._add_proj4_string(datasets, first_dataset, **kwargs)
```

Comment:
You only expect a single keyword argument here, right? Could you pass only the specific one?

Reply:
Man, you are so fast.

Yeah your are correct.